### DISCUSSION TITLE
Would you consider OpenBSD support?

### DISCUSSION THREAD

**Discussion by ghost** [slackhq/nebula#806] [Ideas]
I am a huge fan of OpenBSD having learned Unix-like operating systems on it. Given that OpenBSD has good support for the Go compiler, would there be any interest in creating binaries for it? I am certainly no programmer by any means but I would willingly assist in testing and I would supply a public VM for such purposes. Given OpenBSD's emphasis on security, firewalling, and routing (also having a kernel implementation of WireGuard), I think it would make a great addition to the Nebula family. Please let me know if there is interest. Thank you in advance.

**Comment by johnmaguire** [comment]
@ablackcatstail I've converted this to an issue as it seems to be more of a feature request. Thanks for raising!
